### DISCUSSION TITLE
Lammps SSCHA

### DISCUSSION THREAD

**Discussion by jhung12** [SSCHAcode/python-sscha#130] [Q&A] [answered]
Hi 

In the future, will SSCHA be interfacing with lammps using machine learning potential?

**Comment by diegomartinez2** [comment, marked as answer]
Hi,

SSCHA use ASE for LAMMPS integration, so this is more of a ASE question than SSCHA.

With:
`from ase.calculators.lammpsrun import LAMMPS`
then you can use the ASE calculator in SSCHA.
